src/rdata/mod.rs

```diff
@@ -69,9 +73,11 @@ pub fn fmt_rdata(rtype: ::iana::Rtype, parser: &mut ::bits::Parser,
 pub mod parsed {
     pub use super::rfc1035::parsed::*;
     pub use super::rfc3596::Aaaa;
+    pub use super::rfc2782::Srv;
```

Comment:
Since `Srv` is generic over the domain name type, this needs to be a type alias for `Srv<ParsedDName<'a>>`. We could either define that here or have a sub-module `rfc2782::parsed` just like `rfc1035` has. Since there’s only one record type defined in RFC 2782, I suppose defining a `pub type Srv<'a>` right here is fine.